### PR TITLE
update mapname in kernelnative enhanced

### PR DESCRIPTION
**Pull request by lec-bit**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by lec-bit** [comment]
https://github.com/kmesh-net/kmesh/issues/1099

File: bpf/kmesh/bpf2go/kernelnative/enhanced/kmeshsockops_bpfeb.go [hunk 1]
```diff
@@ -16,34 +16,19 @@ type KmeshSockopsBuf struct{ Data [40]int8 }
...
-			Netns  uint64
-			Ipv4   uint32
-			Port   uint32
-			Family uint32
-			_      [4]byte
-		}
-	}
-}
-
-type KmeshSockopsRatelimitValue struct {
```
Comment (LiZhenCheng9527) [inline bpf/kmesh/bpf2go/kernelnative/enhanced/kmeshsockops_bpfeb.go:42]: Will there be any effect when this part is deleted?

Comment (lec-bit) [reply bpf/kmesh/bpf2go/kernelnative/enhanced/kmeshsockops_bpfeb.go:42]: No effect，Ratelimit is not enable when kernel enhanced

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

Is there a machine available in openEuler OS?

**Pull request merged**